build.wake

```diff
@@ -13,100 +13,103 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package build_wake
 from wake import _
 
 # Useful build variants (arguments to all)
 def toVariant = match _
     "default" = Pass (Pair "native-cpp14-release" "native-c11-release")
-    "static"  = Pass (Pair "native-cpp14-static"  "native-c11-static")
-    "debug"   = Pass (Pair "native-cpp14-debug"   "native-c11-debug")
-    "wasm"    = Pass (Pair "wasm-cpp14-release"   "wasm-c11-release")
+    "static" = Pass (Pair "native-cpp14-static" "native-c11-static")
+    "debug" = Pass (Pair "native-cpp14-debug" "native-c11-debug")
+    "wasm" = Pass (Pair "wasm-cpp14-release" "wasm-c11-release")
```

Comment:
We can't line up the `=` signs in a group of `=`s?

Reply:
discussed in the meeting but doing this inside of `match`es is 100% plan of record. I was just planning on punting to post "MVP" since no info is lost in not aligning today. My hand might be forced here though :)